VideoLocker/src/main/java/org/edx/mobile/player/VideoListFragment.java

```diff
@@ -187,11 +188,18 @@ private void addDataToOnlineAdapter() {
             adapter = new OnlineVideoAdapter(getActivity(), db , storage) {
                 @Override
                 public void onItemClicked(final SectionItemInterface model, final int position) {
+
                     if (model.isDownload()) {
                         // hide delete panel first, so that multiple-tap is blocked
                         hideDeletePanel(VideoListFragment.this.getView());
-
                         DownloadEntry de = (DownloadEntry) model;
+
+                        if ( de.isVideoForWebOnly ){
+                            Toast.makeText(getActivity(), "The video is available for web only", Toast.LENGTH_SHORT).show();
```

Comment:
This needs to be a localized string from a strings file. Also, we should probably get text from docs. cc @lamagnifica